tests/test_validation.py

```diff
@@ -710,7 +710,7 @@ def validate_hypotest(pdf, data, mu_test, expected_result, tolerance=1e-6):
     'setup_and_tolerance',
     [
         (setup_1bin_shapesys(), 1e-6),
-        (setup_1bin_lumi(), 1e-6),
+        (setup_1bin_lumi(), 4e-6),
```

Comment:
Would be good to have a comment in the main body of the PR about why the tolerance needed to be changed here. I don't think we need a full study, just a comment about why it was changed.

Reply:
done